makefile

```diff
@@ -29,6 +29,11 @@ test:
 	python -m coverage run -m attribution.tests
 	python -m coverage report
 	python -m mypy attribution
+	python -m pessimist --fast --requirements=requirements.txt -c "python -m attribution.tests" .
```

Comment:
These requirements don't seem to be adding anything.  If a canon name is in both metadata deps and requirements.txt it will use the constraint from metadata.  See https://github.com/python-packaging/pessimist/blob/main/pessimist/manager.py#L135

Is this future-proofing against some theoretical test-only dep?